app/code/community/Aligent/TimedBanners/Helper/Data.php

```diff
@@ -1,2 +1,58 @@
 <?php
-class Aligent_TimedBanners_Helper_Data extends Mage_Core_Helper_Abstract { }
\ No newline at end of file
+/**
+ * Class Aligent_TimedBanners_Helper_Data
+ */
+class Aligent_TimedBanners_Helper_Data extends Mage_Core_Helper_Abstract {
+    const CONFIG_PREFIX = 'aligent_timedbanners/settings/';
+    const DATETIME_FORMAT = 'd/m/y g:i A';
+
+    /**
+     * @return bool
+     */
+    public function shouldShowBanner()
+    {
+        return Mage::getStoreConfig(self::CONFIG_PREFIX . 'enabled') && $this->isWithinTimedBannerRange();
+    }
+
+    /**
+     * @return string
+     */
+    public function getMessage() {
+        return $this->__(Mage::getStoreConfig(self::CONFIG_PREFIX . 'message', Mage::app()->getStore()));
+    }
+
+    /**
+     * @return DateTime
+     */
+    protected function getStart()
+    {
+        return DateTime::createFromFormat(self::DATETIME_FORMAT, Mage::getStoreConfig(self::CONFIG_PREFIX . 'start'));
```

Comment:
* Are there any concerns with an invalid date appearing in system config causing this function to return false?  Do all consumers of this function allow for booleans to be returned?  
* I have a minor concern that the UI uses locale settings https://github.com/aligent/magento-timed-banners/commit/493e00d6c2e1f424e0fd57c0039758dbaaaf0d67#diff-29ca84d72b5ec0fb6cbdbb7dd0b641ddR13 but the helper hard codes a date format.  Is that a concern?
* Using `DateTime` directly isn't very "Magentoish".  The main side effect of this will be that the banner start/end times will probably be in UTC (since Magento messes with the default timezone), which probably isn't what the client expects.  Could we use the Magento date/time functionality to convert this to a Zend_Date instead?